lib/fireblocks/client.go

```diff
@@ -0,0 +1,105 @@
+package fireblocks
+
+import (
+	"context"
+	"crypto/rsa"
+	"os"
+
+	"github.com/omni-network/omni/lib/apirequestor"
+	"github.com/omni-network/omni/lib/errors"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+const transactionEndpoint string = "v1/transactions"
+
+type FireBlocks interface {
```

Comment:
As mentioned before, the main interface should be named `Client`. You can call the implementation `HTTPClient`?

Reply:
missed this one good catch